src/main/java/io/cdap/plugin/salesforce/plugin/source/batch/SalesforceBaseSourceConfig.java

```diff
@@ -188,6 +219,19 @@ public String getOrgId(OAuthInfo oAuthInfo) throws ConnectionException {
     return partnerConnection.getUserInfo().getOrganizationId();
   }
 
+  public Long getInitialRetryDuration() {
+    return initialRetryDuration == null ? SalesforceSourceConstants.DEFAULT_INITIAL_RETRY_DURATION_SECONDS :
+      initialRetryDuration;
+  }
+
+  public Long getMaxRetryDuration() {
+    return maxRetryDuration == null ? SalesforceSourceConstants.DEFULT_MAX_RETRY_DURATION_SECONDS : maxRetryDuration;
+  }
+
+  public Integer getMaxRetryCount() {
+    return maxRetryCount == null ? SalesforceSourceConstants.DEFAULT_MAX_RETRY_COUNT : maxRetryCount;
+  }
+
```

Comment:
@Shubhangi-cs use can primitive types here.

Reply:
This is fine